screen/edit-transaction/src/main/java/com/ivy/transaction/EditTransactionScreen.kt

```diff
@@ -455,6 +459,9 @@ private fun BoxWithConstraintsScope.UI(
                 // create new mode
                 ModalAdd {
                     onSave(true)
+                    if (toAccount == null && transactionType == TransactionType.TRANSFER) {
```

Comment:
@diasmashikov that check and showing the toast should be done in the ViewModel

Reply:
@diasmashikov if you're up to making it nicer - check https://t.me/c/1647280565/8169/24035

Basically, create a `Toaster` class that encapsulates the logic for displaying toasts